### ISSUE TITLE
Error while loading plugin counter.so does not exists even it exists

### ISSUE THREAD

**Issue by petrkr** [mtester270/archivczskpremium#16]
Hello,
regular version works (as plugin, but not services, they says Error for iPrima for example). So I wanted try premium. But paid version does not work at all. It fails even when try open Plugin window.

It says _counter.so does not exists even that file is there

Here is backtrace

```
02:09:09.6320 Components/PluginComponent.py:56 readPluginList  Plugin  Extensions/archivCZSKpremium failed to load: /usr/lib/python2.7/site-packages/Crypto/Util/_counter.so: cannot open shared object file: No such file or directory                                                                                                         
02:09:09.6324  Traceback (most recent call last):                                                                                                                       
02:09:09.6325    File "/usr/lib/enigma2/python/Components/PluginComponent.py", line 53, in readPluginList                                                               
02:09:09.6326    File "/usr/lib/enigma2/python/Tools/Import.py", line 2, in my_import                                                                                   
02:09:09.6327    File "plugin.py", line 4, in <module>                                                                                                                  
02:09:09.6331    File "archivczsk.py", line 13, in <module>                                                                                                             
02:09:09.6334    File "engine/addon.py", line 7, in <module>                                                                                                            
02:09:09.6337    File "engine/tools/util.py", line 23, in <module>                                                                                                      
02:09:09.6341    File "engine/tools/aes.py", line 2, in <module>                                                                                                        
02:09:09.6344    File "/usr/lib/python2.7/site-packages/Crypto/Random/__init__.py", line 29, in <module>                                                                
02:09:09.6346      from Crypto.Random import _UserFriendlyRNG                                                                                                           
02:09:09.6347    File "/usr/lib/python2.7/site-packages/Crypto/Random/_UserFriendlyRNG.py", line 38, in <module>                                                        
02:09:09.6349      from Crypto.Random.Fortuna import FortunaAccumulator                                                                                                 
02:09:09.6350    File "/usr/lib/python2.7/site-packages/Crypto/Random/Fortuna/FortunaAccumulator.py", line 39, in <module>                                              
02:09:09.6352      import FortunaGenerator                                                                                                                              
02:09:09.6352    File "/usr/lib/python2.7/site-packages/Crypto/Random/Fortuna/FortunaGenerator.py", line 35, in <module>                                                
02:09:09.6355      from Crypto.Util import Counter                                                                                                                      
02:09:09.6355    File "/usr/lib/python2.7/site-packages/Crypto/Util/Counter.py", line 30, in <module>                                                                   
02:09:09.6357      from Crypto.Util import _counter                                                                                                                     
02:09:09.6358  ImportError: /usr/lib/python2.7/site-packages/Crypto/Util/_counter.so: cannot open shared object file: No such file or directory                         
02:09:09.6888 Components/Ipkg.py:28 opkgAddDestination  [Ipkg] Added to OPKG destinations:
```

I am using PurE2 image in version 6.3

Installed it by 
```
opkg install https://github.com/mtester270/archivczskpremium/raw/master/archivczskpremium_1.0-20190503.ipk
```

```
root@sf4008:~# ls /usr/lib/python2.7/site-packages/Crypto/Util/_counter.so -lh
-rw-r--r--    1 root     root       12.5K Aug 11 01:55 /usr/lib/python2.7/site-packages/Crypto/Util/_counter.so
```

screenshot from GUI:
![image](https://user-images.githubusercontent.com/6428351/62828065-93011900-bbdd-11e9-89d9-b70a6d3bd0b3.png)

**Comment by mtester270** [comment]
Read first before ask.
https://github.com/mtester270/archivczskpremium/releases/tag/v1.0.0

opkg install https://github.com/mtester270/archivczskpremium/releases/download/v1.0.0/archivczskpremium-fix-1_1_0.ipk

If you dont understand slovak use google translate. If this does not help you must find python crypto lib for your image (PureE2), i can't help you with this.

**Comment by petrkr** [comment]
Thanks I will try.

I did not read "release notes" as I used other howto from Popup free version. Did not know you have more places where making documentation. I suggest you put this information directly to README or FAQ to main repository. Would be better to find it.

Because I play with that until 2am and I also googled it. But strange was why that file is there, but it says "it is not there".. My guess was some dependency but have no "ldd" command in my box to check all dependencies.

Later today I will try and let you know.

Thanks Petr
